src/config.rs

```diff
@@ -1537,7 +1537,7 @@ impl Config {
 
         if let Some(old_path) = old.as_ref().filter(|old| old.exists()) {
             warn!(
-                "Found configuration at: '{}'. The file should be moved to the new location: '{}'.",
+                "Found configuration at: `{}`. The file should be moved to the new location: `{}`",
```

Comment:
I don't really want to open up a whole new can of worms here, since I think this PR is pretty much good to go, but maybe this isn't too big of a deal:

If we don't have punctuation at the end of our log messages, I wouldn't expect a full stop in the middle of the log message either, since that suggests to me that these are proper sentences will full punctuation. I'd use `;` or `,` here rather than a full stop to resolve this problem. What do you think?